src/main/java/pl/sda/library/infrastructure/jpa/DtoFactory.java

```diff
@@ -59,13 +59,13 @@ public UserDto createUserDto(User user){
         return userDto;
     }
 
-    public AuthorDto createAuthorDto(Author author){
-        AuthorDto authorDto=new AuthorDto();
-        authorDto.setId(author.getId());
-        authorDto.setFirstName(author.getFirstName());
-        authorDto.setLastName(author.getLastName());
-        return authorDto;
-    }
+//    public AuthorDto createAuthorDto(Author author){
```

Comment:
do wyrzucenia

Reply:
tu generalnie cała klasa jest do wyrzucenia (może za wyjątkiem bookDto ale to się okaże)